config/webpack.dev-stage.config.js

```diff
@@ -11,12 +11,18 @@ const PostCssRtlPlugin = require('postcss-rtl');
 
 const commonConfig = require('./webpack.common.config.js');
 const presets = require('../lib/presets');
+const resolvePrivateEnvConfig = require('../lib/resolvePrivateEnvConfig');
 
 // Add process env vars. Currently used only for setting the server port
 dotenv.config({
   path: path.resolve(process.cwd(), '.env.development-stage'),
 });
 
+// Allow private/local overrides of env vars from .env.development for config settings
+// that you'd like to persist locally during development, without the risk of checking
+// in temporary modifications to .env.development.
+resolvePrivateEnvConfig('.env.private');
```

Comment:
do we need it here or  just in webpack.dev.config.js file?

Reply:
Yeah, let's keep it in both configs so that .env.private will apply to either :)